doc/debian_upgrade/SONiC_Debian_Upgrade_Cadence.md

```diff
@@ -0,0 +1,219 @@
+	#### Phase 1
+	* database
+	* swss and orchagent
+	* teamd
+	* pmon
+	* lldp
+	* snmp
+	* syncd/saiserver /syncd-rpc
+	* frr
+	* radvd
+	* nat
+	* eventd
+	* dhcp-relay
+	* telemetry
+	* macsec
+	* sflow
+	* mux
+
+	#### Phase 2
+	* p4rt
+	* gbsyncd
+	* iccpd
+	* restapi
+	* dhcp-server
+	* sonic-sdk
+	* ptf
+	* mgmt-framework
+	* PDE
+ 
+* Phase 1 list container upgrades should be covered in ‘second’ release. It is also recommended to upgrade Phase 2 containers in 'second' release, but will be best effort only.
+	* Ex: Phase 1 list container should be targeted for upgrade to bookworm in 202405 release. Container from phase 2 list can be included in 202405 releases if PR is raised within the time.
+
+<a id="procedure2"></a>
+### Procedure
+
+* Create docker-base-\<debian> and docker-config-engine-\<debian>.
+* Create docker-swss-layer-\<debian>.
+	* orchagent, teamd, frr, nat, sflow are using this swss-layer.
+* Upgrade Dockerfile.j2 of each container to point to latest Debian.
+	* Sometimes this may need packaging updates in some containers.
+
+<a id="timeline2"></a>
+### Timeline
+
+| Task                                        | Time estimate  | Description                       |
+| --------------------------------------------| ---------------|-----------------------------------|
+| Create base docker and config-engine docker | 1 week         |                                   |
+| Create swss-layer docker                    | 1 week         |                                   |
+| Dockerfile migration for each container     | -              | Task for respective owners to upgrade the containers.|
+
+## SONiC Debian support
+This section describes SONiC Debian support process to update SONiC with latest fixes/CVEs from Debian community.
+
+Debian community has three active stable releases named stable, oldstable and oldoldstable([Debian Releases](https://www.debian.org/releases/)). SONiC releases mapping to these Debian releases needs active support. 
+
+This document is targeted at packages which needs manual update in sonic-buildimage. Below is the guidelines and procedure,
+
+<a id="guidelines3"></a>
+### Guidelines 
+
+* Target Debian source which needs active support is currently limited to list below. Minor version of these software package will be updated to the latest available from Debian community. 
+	* Linux Kernel
+* Target timelines for Linux Kernel minor version upgrades on different SONiC branches.
+	* Branches based on Debian 'Stable' - 6 months.
+	* Branches based on Debian 'OldStable' - 6 months.
+	* Branches Based on Debian 'OldOldStable' - Based on requirement.
+		* Ex: As per the current status below is the mapping of SONiC branches to Debian releases.
+			* OldOldStable (Buster) -  202006, 202012, 202106
+			* OldStable (Bullseye) - 202111, 202205, 202211, 202305
+			* Stable (Bookworm) - 202311, master
+* Other third-party list below will be updated based on requirement.

```

Comment:
Can we add `dockerd` to this list as well?

Reply:
done.